README.rst

```diff
@@ -30,15 +30,15 @@ Once you have cookiecutter installed you can run::
   cookiecutter gh:astropy/package-template
 
 
-Which will ask you a series of questions to configure your package. For more information you can see the `package template documentation <>`__.
+Which will ask you a series of questions to configure your package. For more information you can see the `package template documentation <http://docs.astropy.org/en/latest/development/affiliated-packages.html>`__.
 
 
 Improving the package template
```

Comment:
While you're at it updating the readme, could you add a section mentioning the rendered and master branches? I think that would have helped for #244